single-file-document/list_undefined_references.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Can we add this script to the lint Github action?

Reply:
I find this script to be a bit too heavy-weight as it needs all the LaTeX machinery. It also still produces ~180 errors for the NodeJS and Python API docs. I will look into those later. So I'd defer adding this to the lint action now.